app/assets/stylesheets/main.css

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+  font-family: 'Ubuntu', sans-serif;
+}
+.mainContentWrapper {
+  padding: 0;
+  margin: 0;
+}
+.mainContentContainer {
+  max-width: 1200px;
+  margin: 0px auto;
+}
+
+.flexContainerColumn {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.flexBoxColumn {
```

Comment:
do you have a template online just to get an idea of what the overall design is like?